themes/vue/layout/page.ejs

```diff
@@ -34,9 +34,9 @@
   <% } %>
   <div class="footer">
     <%- partial('partials/ad-text') %>
-    Caught a mistake or want to contribute to the documentation?
+    Nếu phát hiện lỗi hoặc muốn góp phần vào nội dung?
     <a href="https://github.com/vuejs/vuejs.org/blob/master/src/<%- page.path.replace(/\.html$/, '.md') %>" target="_blank">
-      Edit this page on Github!
+      Chỉnh sửa trang này trên Github!
```

Comment:
Nên là GitHub thay vì Github vì đó là tên đúng (bản tiếng Anh cũng sai).

Reply:
Oh, I completely did not notice this.

Reply:
Dạ ok anh